PyOpenWorm/contextualize.py

```diff
@@ -272,6 +272,14 @@ def contextualize_class_augment(self, context):
         res.__module__ = self.__module__
         return res
 
+    @property
+    def context(self):
+        return None
+
+    @context.setter
+    def context(self, ctx):
```

Comment:
What's your thinking for adding this here? I would like to be cautious about adding a data descriptor here since that affects how subclasses can be used. For ContextMeta, above, I know how ContextMeta is used internally to PyOpenWorm...I think I would actually rename it to _ContextMeta to signal that it's private.

For ContextualizableClass, it's intended that you can sub-class it as needed, so I want it to provide useful, but conservative, defaults and no more.